CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
 ## [NEXT RELEASE]
 
 ### Added Features
+- ROX-18689: ACS will qualify the registry (and path) of images from the container runtime when env var `ROX_UNQUALIFIED_SEARCH_REGISTRIES` is set to `true` on both Central and Sensor.
```

Comment:
> Do you know if/how this would make it into 4.4.1?

The changelog specifically? No I do not. As I understood it the changelog was one of the sources used to be create the release notes, but not everything in the changelog is used.

Reply:
> nit: ACS will fully qualify the image with this data if it's missing from the deployment configuration

The "if it's missing..." part isn't' technically true, if the registry/repo is populated in the spec and the runtime sets `imageID` to some other registry/repo then an update will be made - in the future if this field becomes representative of mirrors this will 'just start working'.  Central may not display the update however if the registry/repo from the spec yields a successful metadata pull / scan prior to the pod running.